packages/devtools_app/test/flutter/info_screen_test.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/src/connected_app.dart';
+import 'package:devtools_app/src/globals.dart';
+import 'package:devtools_app/src/info/flutter/info_screen.dart';
+import 'package:devtools_app/src/service_manager.dart';
+import 'package:devtools_app/src/vm_service_wrapper.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:vm_service/vm_service.dart';
+
+import 'wrappers.dart';
+
+void main() {
+  InfoScreen screen;
+  group('Info Screen', () {
+    setUp(() {
+      setGlobal(ServiceConnectionManager, MockServiceManager());
+      when(serviceManager.service.getFlagList()).thenAnswer((_) => null);
+      when(serviceManager.connectedApp.isAnyFlutterApp)
+          .thenAnswer((_) => Future.value(true));
+
+      screen = const InfoScreen();
+    });
+
+    void mockFlags() {
+      when(serviceManager.service.getFlagList()).thenAnswer((invocation) {
+        return Future.value(
+          FlagList()
+            ..flags = [
+              Flag()
+                ..name = 'flag 1 name'
+                ..comment = 'flag 1 comment contains some very long text '
```

Comment:
feel free to in the future use
'''Foo bar baz
asdfjkas d
asdfasdfsdf
'''
to enter multiline strings particularly within tests as there is a bit less manual work that way.

Reply:
Noted.